### PR TITLE
[QA-1899] Test failure tool: Save page's content to file when test fails

### PR DESCRIPTION
**Pull request by aweng98**
https://broadworkbench.atlassian.net/browse/QA-1899

Save page's content to a .html file along with screenshot to help with test failure troubleshooting.

### CODE REVIEW

**Comment by nawatts** [comment]
`maybeSaveScreenshot` has a section:
```js
if (screenshotBucket) {
  const storage = new Storage()
  await storage.bucket(screenshotBucket).upload(path)
  if (errorsPresent) {
    await storage.bucket(screenshotBucket).upload(failureNotificationDetailsPath)
  }
}
```

Does `savePageContent` also need something similar?

**Comment by aweng98** [comment]
I think it's fine without it. Is anything (like dashboard) currently using the artifacts saved to the bucket?

**Comment by nawatts** [comment]
> I think it's fine without it. Is anything (like dashboard) currently using the artifacts saved to the bucket?

👍  After some investigation, it looks like the screenshot bucket is a lingering remnant of the Bueller code, which was previously removed. So I don't think that anything is saving screenshots to a bucket.

**Review by nawatts — APPROVED** [review]

File: integration-tests/utils/integration-utils.js [hunk 1]
```diff
@@ -277,11 +277,16 @@ const openError = async page => {
...
     screenshotDirPath :
     process.env.SCREENSHOT_DIR || process.env.LOG_DIR || resolve(__dirname, '../test-results/screenshots')
+  mkdirSync(dir, { recursive: true })
+  return dir
+}
 
+const maybeSaveScreenshot = async (page, testName) => {
+  const dir = getScreenshotDir()
   try {
     mkdirSync(dir, { recursive: true })
```
Comment (cahrens) [inline integration-tests/utils/integration-utils.js]: Shouldn't need this because `getScreenshotDir` makes the directory.

**Review by cahrens — APPROVED** [review]
One minor comment, otherwise LGTM.

File: integration-tests/utils/integration-utils.js (hunk 1 above)
Comment (aweng98) [reply integration-tests/utils/integration-utils.js]: Yeah forgot to remove it. Thanks for catching that.

**Pull request merged**